packages/docs/concepts/lowdefy-schema.yaml

```diff
@@ -24,204 +24,262 @@ _ref:
 
+      - id: yaml_understanding_alert
+        type: Alert
+        properties:
+          type: warning
+          showIcon: false
+          message: |
+            A good understanding of YAML is needed before starting with Lowdefy. If you don't have any experience using YAML, you can find a good introduction video <a href = "https://www.youtube.com/watch?v=cdLNKUoMc6c">here</a>.
+
+      - id: md2
+        type: MarkdownWithCode
+        properties:
+          content: |
             The root schema for a Lowdefy app is:
-            - `lowdefy: string`: __Required__ - The Lowdefy version number the app uses. This is required and cannot be a reference to another file.
+            - `lowdefy: string`: __Required__ - The Lowdefy version number that the app uses. This is required and cannot be a reference to another file.
             - `name: string`: A name for the application.
-            - `license: string`: A [SPDX license ID](https://spdx.org/licenses/). You can use this to indicate the project's  license if you are licencing your project under a specific software license. If you wish to indicate to others that you do not grant the right to use your project, you can optionally use `UNLICENSED` for this field. How you share your Lowdefy config is up to you.
+            - `version: string`: The version number of the app that you are building. This is optional and allows you indicate the version of your app.
+            - `license: string`: A [SPDX license ID](https://spdx.org/licenses/). You can use this to indicate the project's license if you are licensing your project under a specific software license. If you wish to indicate to others that you do not grant the right to use your project, you can optionally use `UNLICENSED` for this field. How you share your Lowdefy config is up to you.
             - `cli: object`: An object with configuration for the CLI.
             - `config: object`: An object with app configuration like the home page pageId.
+            - `auth: object` An object with auth configuration such as providers and an adapter.
             - `global: object`: A data object that can be accessed anywhere in the app using the [`_global`](/_global) operator.
             - `connections: object[]`: An array of [`connection`](/connections-and-requests) objects.
-            - `types: object`: An object to customize and add block types.
+            - `plugins: object[]`: An array of `plugin` objects to customize and add block types.
             - `menus: object[]`: An array of menu objects.
             - `pages: object[]`: An array of page objects.
 
-            # Config
+            Pages are made up of blocks. Blocks have the following basic schema:
+            - `id: string`: __Required__ - A unique identifier for a block.
+            - `type: string`: __Required__ - This is the block type identifier and defines what block to used.
+            - `properties: object`: All the settings passed to a block component.
```

Comment:
Maybe not "All", it can just read: Settings passed to a block React component.
Please also add something like:
- layout: Control how the block will be rendered in the page flow, or in other words, how the is placed in relation to other. See layout for more details.

Reply:
Change has been made in #1314.